xbmc/utils/LangCodeExpander.cpp

```diff
@@ -410,6 +413,24 @@ bool CLangCodeExpander::LookupInUserMap(const std::string& code, std::string& de
   return false;
 }
 
+bool CLangCodeExpander::LookupInLangAddons(const std::string& code, std::string& desc)
+{
+  if (code.empty())
+    return false;
+
+  std::string sCode(code);
+  StringUtils::ToLower(sCode);
+  StringUtils::Trim(sCode);
+  StringUtils::Replace(sCode, '-', '_');
+  std::string langDesc = g_langInfo.GetEnglishLanguageName(sCode);
```

Comment:
const std::string

Reply:
And the whole block until end of function can be simplified to
‘’’
desc = g_lang…;
return !desc.empty();
‘’’

Reply:
done